gui/editor/editor.cpp

```diff
@@ -1001,6 +1001,40 @@ void NumeReEditor::OnChar( wxStyledTextEvent& event )
     if (sAutoCompList.length())
         AutoCompShow(lenEntered, sAutoCompList);
 
+    // if line indicator setting is active, provide autowrapping for comments
+    bool bLineIndicatorSet = m_options->getSetting(SETTING_B_LINELENGTH).active();
+    const int iMaxLineLength = 100;     // member from codeanalyzer.hpp private MAXLINESOFCODE
+    const int iColumnPos = GetColumn(currentPos);
+
+    if (bLineIndicatorSet && (iColumnPos > iMaxLineLength))

```

Comment:
Your solution only works for NumeRe-specific file types. In this case, you want to restrict the solution to those file types by using `m_fileType == FILE_NSCR || m_fileType == FILE_NPRC || m_fileType == FILE_NLYT`